### PR TITLE
Add npm install timeout notification

### PR DESCRIPTION
**Pull request by hardillb**
part of https://github.com/node-red/node-red/issues/4622

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Proposed changes

Adds a notification if the HTTP POST to install a new node is closed by a intermediate proxy (e.g. nginx)

image

## Checklist

- [x] I have read the [contribution guidelines](https://github.com/node-red/node-red/blob/master/CONTRIBUTING.md)
- [ ] For non-bugfix PRs, I have discussed this change on the forum/slack team.
- [ ] I have run `npm run test` to verify the unit tests pass
- [ ] I have added suitable unit tests to cover the new/changed functionality

### CODE REVIEW

**Comment by hardillb** [comment]
I'm not sure why the test is failing, while it is in the node install space, it appears to be testing the backend, not the frontend, so should not be impacted by the changes.

**Comment by hardillb** [comment]
OK, test passed when I rebased the branch on the dev branch

File: packages/node_modules/@node-red/editor-client/locales/en-US/editor.json [hunk 1]
```diff
@@ -643,6 +643,7 @@
             "errors": {
                 "catalogLoadFailed": "<p>Failed to load node catalogue.</p><p>Check the browser console for more information</p>",
                 "installFailed": "<p>Failed to install: __module__</p><p>__message__</p><p>Check the log for more information</p>",
+                "installTimeout": "<p>Install continuing the background, Nodes will appear in palette when complete.</p><p>Or you can monitor the install logs</p>",
```
Comment (knolleary) [inline packages/node_modules/@node-red/editor-client/locales/en-US/editor.json:646]:
```suggestion
                "installTimeout": "<p>Install continuing the background.</p><p>Nodes will appear in palette when complete. Check the log for more information.</p>",
```

I may be wrong, but I don't think we use 'you' in our messages. Have tweaked the wording to be a bit more inline with the other messages.

**Review by knolleary — CHANGES_REQUESTED** [review]
One small wording tweak, but otherwise, looks good.

**Review by knolleary — APPROVED** [review]

**Pull request merged**